.github/workflows/lint.yml

```
@@ -0,0 +1,20 @@
+name: lint proyect
+
+on:
+    push: 
+        branches: [ main ]
+
```

```suggestion
on:
    push: 
        branches: [ main ]
  pull_request:
    branches: [main]
  workflow_dispatch:

permissions:
  contents: write
```

Para que el workflow se ejecute al abrir una PR

---

Error mio, no estan bien alineados, en los comentarios parecia que si 😅